src/WebJobs.Extensions.DurableTask/HttpManagementPayload.cs

```diff
@@ -90,5 +90,63 @@ public class HttpManagementPayload
         /// </value>
         [JsonProperty("resumePostUri")]
         public string ResumePostUri { get; internal set; }
+
+        /// <summary>
+        /// Gets or sets the forwarded host header value.
+        /// </summary>
+        /// <value>
+        /// The forwarded host header value.
+        /// </value>
+        [JsonProperty("forwardedHost")]
+        public string ForwardedHost { get; internal set; }
```

Comment:
It doesn't look like these changes are being used, so I think we can safely remove them from this PR.